src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/Codes/Codes.tsx

```diff
@@ -118,15 +118,17 @@ const Codes = ({
       >
         Custom
       </MenuItem>,
-      ...Object.entries(codeSystems).map(([oid, name]) => (
-        <MenuItem
-          key={oid}
-          value={name as string}
-          data-testid={`code-system-option-${name}`}
-        >
-          {name}
-        </MenuItem>
-      )),
+      ...Object.entries(codeSystems)
```

Comment:
Should `relatedCodeConcepts` be also sorted ?

Reply:
It is sorted by codes by default, which the POs have said is good